R/FilterState.R

```diff
@@ -602,7 +614,9 @@ FilterState <- R6::R6Class( # nolint
       if (!is.null(private$cache)) {
         state <- private$cache
         state$disabled <- NULL
+        private$fixed <- FALSE
```

Comment:
I'm confused by this. Why do we set `fixed = FALSE` and then `fixed = TRUE`? I thought fixed was supposed to remain constant.

Reply:
This is so that `set_state` can proceed. Normally it aborts if `fixed = TRUE`.

I agree this it not very neat and I am open to discussion.